### PR TITLE
Correct spelling of to-do list

### LINKED ISSUE #29667: Please rectify Type to follow menu
### Feedback

Please see [here](https://i.imgur.com/X7JYut3.png) 

### URL

https://www.home-assistant.io/integrations/todo/

### Version

2023.11.0

### Additional information

Noting major just house keeping looks rushed

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Make sure we are consistent with how we write to-do lists.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #29667

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/todo.markdown [hunk 1]
```diff
@@ -11,32 +11,32 @@ ha_codeowners:
...
 
-Some To-do List integrations allow Home Assistant to manage the To-do Items in the list. The
-services provided by some To-do List entities are described below or you can read more about [Service Calls](/docs/scripts/service-calls/).
+Some to-do list integrations allow Home Assistant to manage the to-do items in the list. The
+services provided by some to-do list entities are described below or you can read more about [Service Calls](/docs/scripts/service-calls/).
 
 ### Service `todo.add_item`
 
-Add a new To-do Item. A To-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector) and the `data` payload supports the following fields:
+Add a new to-do Item. A to-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector) and the `data` payload supports the following fields:
```
Comment (MartinHjelmare) [inline source/_integrations/todo.markdown]:
```suggestion
Add a new to-do item. A to-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector) and the `data` payload supports the following fields:
```

File: source/_integrations/todo.markdown [hunk 2]
```diff
@@ -77,7 +77,7 @@ data:
 
 ### Service `todo.remove_item`
 
-Removing a To-do Item. A to-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector), and the `data` payload supports the following fields:
+Removing a to-do Item. A to-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector), and the `data` payload supports the following fields:
```
Comment (MartinHjelmare) [inline source/_integrations/todo.markdown]:
```suggestion
Removing a to-do item. A to-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector), and the `data` payload supports the following fields:
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
How will this solve the linked issue? Wasn't that case inverted compared to this change?

**Comment by frenck** [comment]
The spelling was inconsistent. It is now consistent with our user interface and according to the recommendations from the Microsoft Style Guide for writing (which we follow).

**Comment by MartinHjelmare** [comment]
Ok. But the case style in the frontend is not consistent between the different items in the side bar.

**Comment by frenck** [comment]
CleanShot 2023-11-10 at 09 29 26@2x

This item in the frontend is correct, as in, follows sentence casing. 

CleanShot 2023-11-10 at 09 30 16@2x

These seem to do so as well? What am I missing?

../Frenck

**Comment by MartinHjelmare** [comment]
That looks good. The screenshot in the issue still had tools in Developer tools capitalized.

**Pull request merged**